shapely/creation.py

```diff
@@ -747,7 +747,7 @@ def prepare(geometry, **kwargs):
     it is efficient to call this function on an array that partially contains
     prepared geometries.
 
-    This function does not return any values; geometries are modified in place.
+    This function returns True where geometries were prepared, and False otherwise.
```

Comment:
Should we stress that the function always succeeds regardless of the return value (unless it would raise an error)?